x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -52,6 +52,17 @@ export const TRACK_CURRENT_LAYER_STATE = 'TRACK_CURRENT_LAYER_STATE';
 export const ROLLBACK_TO_TRACKED_LAYER_STATE = 'ROLLBACK_TO_TRACKED_LAYER_STATE';
 export const REMOVE_TRACKED_LAYER_STATE = 'REMOVE_TRACKED_LAYER_STATE';
 export const SET_TOOLTIP_STATE = 'SET_TOOLTIP_STATE';
+export const UPDATE_DRAW_STATE = 'UPDATE_DRAWMODE';
+
+export const DRAW_STATE_TYPE = {
```

Comment:
Instead of DRAW_STATE_TYPE, why don't you just have a boolean flag, like `isDrawingFilter`?

Its really just an on/off switch. Comparing to a string value fells heavy

Reply:
ok, just removed it altogether